randovania/games/dread/json_data/header.json

```diff
@@ -1406,17 +1411,46 @@
                 }
             },
             "Push Wide Beam Block": {
-                "type": "and",
+                "type": "or",
                 "data": {
                     "comment": null,
                     "items": [
                         {
-                            "type": "template",
-                            "data": "Shoot Charge Beam"
+                            "type": "and",
+                            "data": {
+                                "comment": null,
+                                "items": [
+                                    {
+                                        "type": "template",
+                                        "data": "Shoot Charge Beam"
+                                    },
+                                    {
+                                        "type": "template",
+                                        "data": "Shoot Wide Beam"
+                                    }
+                                ]
+                            }
                         },
                         {
-                            "type": "template",
-                            "data": "Shoot Wide Beam"
+                            "type": "and",
```

Comment:
Wouldn´t this mean that if you activate diffusion abuse, the resolver thinks you can use it on every Wide Beam Block? But we have blocks where it is impossible to use the trick